plugin/adsk/scripts/mayaUSDRegisterStrings.py

```diff
@@ -36,6 +36,6 @@ def mayaUSDRegisterStrings():
     register("kLoadUSDFile", "Load USD File")
     register("kFileOptions", "File Options")
     register("kMakePathRelativeToSceneFile", "Make Path Relative to Scene File")
-    register("kMakePathRelativeToSceneFileAnn", "If enabled, path will be relative to your Maya scene file. If this option is disabled, there is no Maya scene file and the path will be absolute. Save your Maya scene file to disk to make this option available.")
+    register("kMakePathRelativeToSceneFileAnn", "If enabled, path will be relative to your Maya scene file.\nIf this option is disabled, there is no Maya scene file and the path will be absolute.\nSave your Maya scene file to disk to make this option available.")
```

Comment:
Why this change? Were you asked by documentation to make it? I originally added this string exactly as given to me by documentation and QA verified it.

Reply:
The only change is to split it on two line. It seemed hard to read when all on a single very long line: the tooltip was almost as wide as my monitor. I did not think the line break, splitting each sentence on its own line would be important.

Reply:
Confirming: we asked the designers and they are OK with splitting the lines.